cpp/test/Ice/span/Test.ice

```diff
@@ -12,31 +12,31 @@ module Test
 
     interface TestIntf
```

Comment:
Why remove the metadata from the `out` parameters in this file, but not `TestAMD.ice`?
This file is used by both the client side and server side (non-amd).

Reply:
view-type has no effect on out parameters, except with AMD.